src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -32,6 +34,7 @@
     public static final String VALID_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.bat";
     public static final String VALID_MAC_FILE_ADDRESS_CS2101 = "./src/test/java/seedu/address/testutil/cs2101.sh";
     public static final String VALID_MAC_FILE_ADDRESS_TESTFILE = "./src/test/java/seedu/address/testutil/testFile.sh";
+    public static final String VALID_CHILD_FOLDER = "child";
```

Comment:
Will this be really valid?

Reply:
I'm just using it as cd user input, but yeah I think the name is kinda vague. I'll change it to something that the project has!